package.json

```
@@ -2,7 +2,7 @@
   "name": "controls-for-yt-shorts",
   "scripts": {
     "dev": "npm-run-all --parallel scss web-ext",
-    "web-ext": "node .ext.mjs",
+    "web-ext": "node .ext-template.mjs",
```

The `.ext-template.mjs` file was meant to be used as a template for the `.ext-mjs`, where the developer rename the file `.ext-template.mjs` to `.ext.mjs` (my bad I should made it clear). 

As the `.ext-mjs` is ignored by git, every developer can have a different testing environment for the extension without the risk pushing changes to it and overwriting the one of other developers. 

```suggestion
    "web-ext": "node .ext.mjs",
```

---

Might make sense to add a postinstall step that creates the file automatically if it doesnt exist.

---

I just added that as well.